docs/pages/Troubleshooting.md

```diff
@@ -8,3 +8,152 @@ You need to delete the HSTS mapping in [chrome://net-internals/#hsts](chrome://n
+
+The following is a walk-through on how to compare the performance impact of an imaginary change in UI5 Builder project.
+
+### Setup
+
+1. Install [hyperfine](https://github.com/sharkdp/hyperfine#installation)
+1. Prepare the UI5 Tooling projects you want to measure *(optional if your development environment already reflects this)*:
+    1. Start in an empty directory
+        ```sh
+        mkdir ui5-tooling-benchmark && cd ui5-tooling-benchmark/
+        ```
+    1. Clone [UI5 CLI](https://github.com/SAP/ui5-cli)
+        ```sh
+        git clone git@github.com:SAP/ui5-cli.git
+        ```
+    1. Clone [UI5 Builder](https://github.com/SAP/ui5-builder) (or your fork!)
+        ```sh
+        git clone git@github.com:SAP/ui5-builder.git
+        ```
+        Make sure you checkout the `master` branch since we will perform the baseline test first
+    1. Install npm dependencies in both directories
+        ```sh
+        (cd ui5-cli && npm install)
+        (cd ui5-builder && npm install)
+        ```
+    1. Create global npm links for both projects
+        ```sh
+        (cd ui5-cli && npm link)
+        (cd ui5-builder && npm link)
+        ```
+    1. Link UI5 Builder into UI5 CLI
+        ```sh
+        (cd ui5-cli && npm link @ui5/builder)
+        ```
+    1. Verify your setup
+        ```sh
+        ui5 --version
+        ```
+        This should output the version and location of the UI5 CLI you just cloned.
+
+        For example:
+        ```
+        2.6.6 (from /my/home/ui5-tooling-benchmark/ui5-cli/bin/ui5.js)
+        ```
+
+1. Prepare your test project (we chose the [openui5-sample-app](https://github.com/SAP/openui5-sample-app))
+    1. Clone the project
+        ```sh
+        git clone git@github.com:SAP/openui5-sample-app.git
+        ```
+    1. Navigate into the project
+        ```sh
+        cd openui5-sample-app
+        ```
+    1. Install any required npm dependencies
+        ```sh
+        npm install
+        ```
+        Note: We will not link UI5 CLI into the but call the global install directly
+    1. Verify that the previously installed UI5 CLI can be called with the following command
+        ```sh
+        UI5_CLI_NO_LOCAL=X node /my/home/ui5-tooling-benchmark/ui5-cli/bin/ui5.js --version
+        ```
+        On Windows:
+        ```sh
+        set UI5_CLI_NO_LOCAL=X node /my/home/ui5-tooling-benchmark/ui5-cli/bin/ui5.js --version
+        ```
+        *(Replace the path to ui5.js with the one shown in the previous `ui5 --version` output)*
+
+### Benchmark
+
+1. Depending on how reliable you want the measurements, consider preparing your system:
+    1. Connect your computer to a power supply
+    1. Make sure no updates or anti virus scans are taking place
+    1. Close all applications. This includes your IDE since it might start indexing any new files created during the build, thus impacting I/O
```

Comment:
```suggestion
    1. Close all applications. This includes your IDE, since it might start indexing any new files created during the build, thus impacting I/O
```